### PR TITLE
tests: use oem instead of hwe kernel for hybrid-model test

### PR DESCRIPTION
**Pull request by sergiocazzolato**
Hybrid-remodel test is failing and it is because the hwe kernel is broken.

### CODE REVIEW

File: tests/lib/assertions/classic-model-rev1-22.assert [hunk 1]
```diff
@@ -38,13 +38,13 @@ snaps:
...
+AcLBcwQAAQoAHRYhBGESvKlz1RXG1IBOC0MdJKf2hr9ABQJm2xVIAAoJEEMdJKf2hr9AFZcP/iNS
+P0VYXtrYRP4SXJnSws9MSvosF1Q6Zj4vKAddKAlswnu87/e6KaKw/xlFg5MfcgoWAjruU662sxPJ
+1X1f579LnPFm7OwnG5IiGKeK5LOtn1T9JADNwLYKQyXlwfiLaBaqPQ0PsXSYxx/v3AW7421sHK2V
+fHtUWSag/olAW+LqerRJsrgSg4UjxpItboej9wsOWzU6rCW9nDlSP4d3FsaYTlasurXUVegtW+XE
+UDdcQ07KLkpJk5PNIa9Ch55gwnMRRhI9eXhibh1L9xDQLtRigzxWQh7W8F08/WoFsywmLdihfUUM
+JUgT7z7t6MmahYVUy3vz7Fhtj5UZCET4UL89/NQ+XUVvjYcqBz0GRFXJ02yqVQNwBe8BvC8I1GQ9
+eQao+hEuJ2Zz51GV1IVkV0VITgo0APWqnF8DHAvTNlJ3rlViJHrbvW7qLxaIvv061Rj9IGE/2RNk
+nW/WZQw5XGPYLz5q00dKiW5/5yTcHkFFiCZBcJws2vKVtZdgr7ggkbApl3UIbWNuTc9Fz6UniVj5
+0MFFGS38bvfngTrHw9fn00zzgtaNGpsN9JmK3baC1qOU9c159vP3nlNkaDe21mpRHU6BX2tacCM9
+nQJetYmBebuUHXq/wl/T6fJNvQk42+u6b+68kx/KAl3udwfsiUF48WiRsHcChqNZQhpRsoCm
```
Comment (andrewphelpsj) [inline tests/lib/assertions/classic-model-rev1-22.assert:50]: I believe assertions are supposed to have this newline here, not sure if that will prevent things from working or not.

File: tests/lib/assertions/classic-model-rev1-24.assert [hunk 2]
```diff
@@ -38,13 +38,13 @@ snaps:
...
+AcLBcwQAAQoAHRYhBGESvKlz1RXG1IBOC0MdJKf2hr9ABQJm2xVeAAoJEEMdJKf2hr9AqyEQAIo9
+VXlntZZGLGO6zc+9z8DINfJkmjgOfmiZ01skJLPk4eP9SszLGodG+1XJozMekgIIGD4rpU2er3rO
+l9IFj5Q9lPHVXGtI2LIIaNMpqf7gijXZ5uY0Sjqh0po4G8XLjUxpDz+WbDNTX3tXXMAhDjFYmSWr
+mjbyFRp7SD0gu7mbdxTO4owobm9hSCZdCryLaw35N39e/MQYaxOLS0a/WllrewRniRAB1o3AQEMw
+8n1biVE9THLjeAIkaFywv5LjCKQFHb3NvfqTG3DKdqLsd8JoB7i6qFXn9Z4sEJVgNCrRm6VfuoRv
+SroeINjJQ09913gf7q+GAtv9Cuh5fnr8VRO1jr343UIRyxAHybsU4ebXpWwVRf0fUn2B+e9xV9rd
+eFFgyCk1C+sJWmVCsHLZez2g1PJdl6shn/dw73PZ7aPHPr9nyikgPKgDCyNWyzfrjHyVOibtMyRQ
+Gr4DL4w2mmoqoTnk7gsOJB61Ros5A/vrwwMzs573V3xoJgbMVAOdXhQ896hqbWdVLqECa+V/PO5X
+oKlZWWqi970nRtF/Xj0/6IYKxrZY1uYF3Nwpr9YOReGGOZE6T+nfVOwga7hBxKLDTNrK06PDuOA3
+euoe0EugTbYxhICfzNYTR9y+I+3Ggk4+Hg7Rlw+RR8lwFVTX8qfSDROs1ckl02mdRNczMFrp
```
Comment (andrewphelpsj) [inline tests/lib/assertions/classic-model-rev1-24.assert:50]: Same here

**Review by andrewphelpsj — APPROVED** [review]
Thanks! Unsure if those newlines matter, but may want to make sure.

File: tests/lib/assertions/classic-model-rev1-22.assert [hunk 3]
```diff
@@ -17,7 +17,7 @@ snaps:
     name: pc
     type: gadget
   -
-    default-channel: 22-hwe/stable
+    default-channel: 22-oem/stable
```
Comment (alfonsosanchezbeato) [inline tests/lib/assertions/classic-model-rev1-22.assert:20]: Nice! Hopefully we are more lucky with this one...

**Review by alfonsosanchezbeato — APPROVED** [review]
LGTM, thanks (but yes, assertions usually have a newline at the end)

**Pull request merged**